src/stmt.rs

```diff
@@ -48,6 +50,7 @@ pub enum Statement {
     Redirect(Option<NAT>),   // redirect is subset of NAT options
     Reject(Option<Reject>),
     Set(Set),
+    // TODO: map
```

Comment:
Hmm, should these TODOs stay in the codebase? Or should we maybe rather create an issue with a task list for these? (Or both?)

Reply:
I prefer to keep the marks in the code base for future code readers to be aware of this.

Created an issue: #70.